shared/cfg/machines.cfg

```diff
@@ -60,6 +60,24 @@ variants:
         usb_devices =
     - s390-virtio:
         only s390x
+        # Usb subsystem not available
+        no usb, usb_device_check, boot_from_usb
+        # Only virtio nic is available
+        no nic_8139, nic_e1000
+        # Suspend (to mem) is not supported
+        no guest_s3, guest_s3_time_drift
+        # Hibernate (to disk) is not yet supported by kernel
+        no guest_s4, guest_s4_time_drift
+        # This is virtio target, pci is not supported
+        no pci_bridge
+        # Memory subsystem is way different and hotplug is not to be supported any time soon
+        no hotplug_memory, hotplug_memory_repeat
+        # NMIs are only produced by machine checks
+        no nmi_watchdog
+        # No sound cards
+        no audio
+        # Only tod clock (timerdevice requires kvm-clock)
+        no timerdevice
```

Comment:
Nice! Although I won't test this, I do trust this is true and find it a welcome addition to new users attempting to use Avocado-VT in such a platform.  Just thinking of how many hours this "small" commit cost! ;)

Reply:
yep, it's not ideal, but having the main filters in machines should be easy to follow and update when features are introduced.

Reply:
look good to me for the devices in the unsupported device list, that's good
Acked-by: zhengtli@redhat.com